src/app.js

```diff
@@ -171,3 +174,7 @@ function getFilterConfig () {
     filename, 'getFilterConfig', 'config string', configString))
   return configString
 }
+
+function getMetaFilterConfig () {
```

Comment:
I think this function could be dropped. Could getFilterConfig be used to get all the filter configurations?

Reply:
a skeleton implementation of the function which reads the metadata filter config values will be a good next step.